src/js/queries/informationPageFragment.js

```diff
@@ -51,6 +51,15 @@ const informationPageFragment = `
           contact {
             name
             email
+            phoneNumber {
```

Comment:
Why not use 
https://github.com/cityofaustin/janis/blob/d577f67d0c0f12c810aedb18a2bf56ec55d62532/src/js/queries/phoneFragment.js ?

Reply:
i copied from guidepage, I've updated the PR so both info page and guide page use the fragment now